system_files/desktop/shared/usr/bin/bazzite-hardware-setup

```diff
@@ -62,6 +62,9 @@ if [[ ":Jupiter:" =~ ":$SYS_ID:"  ]]; then
   if [[ ! $KARGS =~ "nmi_watchdog" ]]; then
     NEEDED_KARGS="$NEEDED_KARGS --append=nmi_watchdog=0"
   fi
+  if [[ ! $KARGS =~ "modprobe.blacklist=sp5100_tco" ]]; then
+    NEEDED_KARGS="$NEEDED_KARGS --append=modprobe.blacklist=sp5100_tco"
```

Comment:
@EyeCantCU I think now this interferes with your implementation at https://github.com/ublue-os/bazzite/commit/7fc18d3c209e6b8af90dc77156477dfaa5dfd8cd
```
# bazzite-hardware-setup
Current kargs: amd_pstate=active amd_iommu=off amdgpu.gttsize=8128 spi_amd.speed_dev=1 audit=0 initcall_blacklist=simpledrm_platform_driver_init rd.luks.options=discard rhgb quiet root=UUID=77ba60d1-1b8b-4f37-b7fd-7eb303d7ec77 rootflags=subvol=root rw ostree=/ostree/boot.0/default/48d2b7a556f23bdd0018f8f6f8c28c2b9b8df22231bb134bc568ff959da39267/0 nowatchdog nmi_watchdog=0 modprobe.blacklist=sp5100_tco
Checking for needed karg changes (Jupiter)
Checking for needed karg changes
Found needed karg changes, applying the following:  --delete=modprobe.blacklist=nouveau
error: No karg 'modprobe.blacklist=nouveau' found
```

Reply:
@KyleGospo